docker-compose.yml

```diff
@@ -1,8 +1,7 @@
 server:
   build: .
   volumes:
-    - .:/app
-    - /app/node_modules
+    - ./conf.js:/app/conf.js
```

Comment:
No, I agree. For development purposes I would probably mount in more directories. I would even consider running the environment without docker container.
I could also make a `docker-compose.development.yml` witch would override the main `docker-compose.yml` with extra volume mappings.